### PR TITLE
Feature/classify videos v1

### PR DESCRIPTION
**Pull request by tomasheiskanen**
Concatenates prompts of all videos and runs batch inference thereafter. The results are saved to csv file in same format as other results.

### CODE REVIEW

File: classify_videos.py [hunk 1]
```diff
@@ -86,25 +86,34 @@
...
-        "\nUser:",
-        image1,
-        image2,
-        "These are a few screenshots from a tiktok video.  As these are just a few screenshots, please do not assume that the video is, for example, a collage.  Please tell me in a few words each what the topic and style of the video might be.",
-        "<end_of_utterance>",
-        "\nAssistant:",
-    ],
-]
+    PROMPT = """
+    These are a few frames from a tiktok video. Write a one paragraph description of the video.
```
Comment (jmccrosky) [inline classify_videos.py:99]: Does it not work better if you explicitly tell it how many frames you're providing?

File: classify_videos.py [hunk 2]
```diff
@@ -86,25 +86,34 @@
...
+    """
+
+    current_frames = selected_frames[video]
+    image1 = Image.open(f"frames/frame_{current_frames[0]}.jpg")
+    image2 = Image.open(f"frames/frame_{current_frames[0]}.jpg")
```
Comment (jmccrosky) [inline classify_videos.py:104]: I think you're showing it the first frame twice.

**Review by jmccrosky — CHANGES_REQUESTED** [review]

File: classify_videos.py (hunk 2 above)
Comment (tomasheiskanen) [reply classify_videos.py:104]: Good catch

File: classify_videos.py (hunk 1 above)
Comment (tomasheiskanen) [reply classify_videos.py:99]: Let's try

Comment (tomasheiskanen) [reply classify_videos.py]: Seems to improve

File: classify_videos.py (hunk 2 above)
Comment (tomasheiskanen) [reply classify_videos.py]: Fixed

**Review by jmccrosky — APPROVED** [review]

**Pull request merged**